apps/testing-javascript/src/styles/code.css

```diff
@@ -0,0 +1,6 @@
+[data-video-code] {
+  @apply mx-auto max-w-4xl p-5 py-16;
+  [data-title] {
+    @apply flex items-baseline text-2xl font-semibold sm:text-3xl;
+  }
+}
```

Comment:
I'm not seeing these get applied to the Code heading, not sure what I'm missing. @DrShpongle can you help me out with this?